chart/kubeapps/templates/frontend/configmap.yaml

```diff
@@ -139,6 +139,22 @@ data:
     {{- end }}
       }
 
+    {{- if .Values.featureFlags.kubeappsAPIsServer }}
+      location ~* /api/kubeappsapis {
```

Comment:
Could we just use `/kubeappsapis` or even `/apis` given that the full paths would then be things like `/apis/core/packages/v1alpha1/...`

What do you think? It's still behind a featureflag so I've no problem changing it later.

Reply:
Good idea indeed, shorter and more concise. Changing right away!
Regarding the SA flag, yep, totally agree. Of course, we are not gonna release a new official version as it is right now,
However, let's see how this flag can be passed through, I'm struggling with passing information towards the plugins. I Will let you know in another PR.